src/mount_style.rs

```diff
@@ -1,37 +1,32 @@
-/// Thanks a lot to thaw ui for this :)
+use cfg_if::cfg_if;
 pub fn mount_style(id: &str, content: &'static str) {
-    #[cfg(feature = "ssr")]
-    {
-        use leptos::html::style;
-        use leptos_meta::use_head;
-        let meta = use_head();
-        let style_el = style()
-            .attr("csr-id", format!("leptos-toaster-{id}"))
-            .child(content);
-        meta.tags
-            .register(format!("leptos-toaster-{id}").into(), style_el.into_any());
-    }
-
-    #[cfg(not(feature = "ssr"))]
-    {
-        use leptos::document;
-        let head = document().head().expect("head no exist");
-        let style = head
-            .query_selector(&format!("style[csr-id=\"leptos-toaster-{id}\"]"))
-            .expect("query style element error");
+    let id = format!("leptos-color-id-{id}");
+    cfg_if! {
```

Comment:
Would prefer to leave out the extra dependency, since I don't think this is better. But not a blocker

Reply:
Nevermind, will merge anyways ;)